test/Servant/QuickCheck/InternalSpec.hs

```diff
@@ -119,6 +135,11 @@ type API = ReqBody '[JSON] String :> Post '[JSON] String
 api :: Proxy API
 api = Proxy
 
+type ParamsAPI = QueryParam "one" String :> QueryParam "two" String :> Get '[JSON] String
```

Comment:
Great idea! I didn't even think you could use Unit actually, and yes I didn't love those assertions much either. I'll give that a go now